### PR TITLE
#4287 - Fix viewDraft ES filter, refs #4287

### PR DESCRIPTION
**Pull request by sevein**
Edgar has sent us a patch that he attached to the corresponding Redmine ticket (https://projects.artefactual.com/issues/4287#change-30595). I'm opening this pull request for him as he couldn't make it at the moment.

### CODE REVIEW

File: plugins/qbAclPlugin/lib/QubitAclSearch.class.php [hunk 1]
```diff
@@ -167,32 +167,39 @@ public static function filterDrafts(\Elastica\Filter\Bool $filterBool)
...
     {
+      // At least one specific rule
       // Get last rule in list, it will be the global rule with the opposite
       // access of the preceeding rules (e.g. if last rule is "DENY ALL" then
       // preceeding rules will be "ALLOW" rules)
       $globalRule = array_pop($repositoryViewDrafts);
 
       $filter = new \Elastica\Filter\Bool();
+      $filter->addShould(new \Elastica\Filter\Term(array('publicationStatusId' => QubitTerm::PUBLICATION_STATUS_PUBLISHED_ID,
+                                                         'publicationStatusId' => QubitTerm::PUBLICATION_STATUS_PRIVATE_ID)));
```
Comment (jraddaoui) [inline plugins/qbAclPlugin/lib/QubitAclSearch.class.php:193]: I think "Private" is not a default publication status in AtoM.

**Comment by jraddaoui** [comment]
I have not been able to reproduce the related issue using qa/2.2.x.

**Comment by sevein** [comment]
@edgarrodsil :pager:

File: plugins/qbAclPlugin/lib/QubitAclSearch.class.php (hunk 1 above)
Comment (edgarrodsil) [inline plugins/qbAclPlugin/lib/QubitAclSearch.class.php:193]: The "Private" status is a customization, I forgot remove it before send the code

**Comment by jraddaoui** [comment]
Looks like this was working in the end or it has been already fixed.

Thanks @edgarrodsil!

**Pull request closed without merging**